client/app/bundles/course/assessment/submission/constants.ts

```diff
@@ -16,7 +16,7 @@ export const questionTypes = mirrorCreator([
 
 export const MEGABYTES_TO_BYTES = 1024 * 1024;
 
-export const BUFFER_TIME_TO_FORCE_SUBMIT_MS = 2 * 60 * 1000;
+export const BUFFER_TIME_TO_FORCE_SUBMIT_MS = 5 * 1000;
```

Comment:
understood. The `timedAssessmentDialogMessage` has been changed, however `timedExamDialogMessage` need not be changed

Reply:
Any reason to not finalize immediately when timer hits 0 though? (i.e. 0 buffer)  
Open for discussion for rationale

Reply:
we haven't had the good UX design for having 0 buffer (for this to happen, we at least need to alert the student if the timer is due, so that they won't be shocked when the timer is suddenly running out and then all their submission gets submitted).

With having buffer, at least the UI for the timer banner will change its color and student will be at least aware that their timer is up, so it won't be much of a shock to them